tedana/decomposition/ica.py

```diff
@@ -64,16 +66,16 @@ def tedica(data, n_components, fixed_seed, maxit=500, maxrestart=10):
 
             w = list(filter(lambda i: issubclass(i.category, UserWarning), w))
             if len(w):
-                LGR.warning('ICA attempt {0} failed to converge after {1} '
-                            'iterations'.format(i_attempt + 1, ica.n_iter_))
+                LGR.warning('ICA with random seed {0} failed to converge after {1} '
```

Comment:
Since we're updating this user-facing behavior, we should make sure it's noted somewhere (maybe in the release notes ?).

Reply:
I think release notes should be okay for this. It's not a significant change.

Reply:
I know when I used to run tedana mors I would watch the logs, so I worry it could scare those users (is why I do want it mentioned somewhere). I know it would have scared me ! 😅

Reply:
That's completely fair. Any thoughts on where to put it aside from release notes? When we do next release we could package it with the newsletter, too.

Reply:
We could downgrade this from a warning to a debug in a separate PR?

Reply:
In general I was thinking that we don't have a descriptive `whatsnew`. Maybe we could consider starting one ?

For now, I think the release notes are fine ! And I always like the newsletter's summaries :)

Reply:
I actually disagree with downgrading to a debug. I've noticed that failure to converge tends to go along with lower-quality data, so I think it's good to let the user know that the software is struggling a bit.
Would you mind opening an issue for a `whatsnew` @emdupre ? That sounds like a good idea to me, especially if we're planning on increasing the release speed.